java/cuvs-java/pom.xml

```diff
@@ -124,7 +126,7 @@
                     </execution>
                     <execution>
                         <id>compile-java-22</id>
-                        <phase>compile</phase>
+                        <phase>process-classes</phase>
```

Comment:
I think this could be very problematic: this change breaks IntelliJ and makes very difficult to navigate code and use the IDE. 
Why was this changed?